### ISSUE TITLE
README documentation: add brackets for project names

### ISSUE THREAD

**Issue by jonathan-gruber** [snyk-tech-services/snyk-scm-contributors-count#65]
the document does not mention that when there are spaces in a project name, brackets would be required
worth adding to the readme

**Comment by lili2311** [comment]
@jonathan-gruber please provide an example?

**Comment by jonathan-gruber** [comment]
yes so yesterday we were on a troubleshooting call and the project name was called "project ABC" with a space.
Took us time to catch this was the reason the script was not running properly until we realized we needed to add brackets
maybe it is too obvious or trivial to add to it in Readme but wanted to share this feedback
